### DISCUSSION TITLE
is it working without buttons mod ?

### DISCUSSION THREAD

**Discussion by tiretrak** [marcel-licence/esp32_drum_computer#11] [Q&A]
i have esp32 audio kit v2.2 .
do i need button mod or i must change code ?

**Comment by marcel-licence** [comment]
You can use the project without the mod. The mod might be required if you want to use all buttons to control different functions. I do not know if there would be any IO conflict with a button and for example MIDI Rx, I2C of the codec etc. It depends on your type of ESP32 Audio Kit. I've seen different configurations / variations of the IO assignments.
